tests_integration/test_data/projects/streamlit/snowflake.yml

```diff
@@ -0,0 +1,6 @@
+definition_version: 1
+streamlit:
+  name: test_streamlit_deploy_snowcli
+  file: streamlit.py
+  stage: streamlit
+  use_packaging_workaround: false
```

Comment:
Wasn't this argument dropped?

Reply:
I thought I removed all of them, thanks!